Examples/Scripts/Python/simulate_barrel.py

```
@@ -0,0 +1,90 @@
+    DetectorBuilder,
+    GeometryIdGenerator,
+)
+from acts import geomodel as gm
+from acts import examples
+
+def runGeant4(
+    detector,
+    trackingGeometry,
+    field,
+    outputDir,
+    materialMappings=[],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)